gensim/models/fast_line_sentence.cpp

```diff
@@ -0,0 +1,29 @@
+#include "fast_line_sentence.h"
```

Comment:
what's about glue `.cpp` and `.h` to one file?

Reply:
done